scripts/kata-doc-to-script.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+[ -n "$DEBUG" ] && set -x
+
+script_name="${0##*/}"
+
+typeset -r warning="WARNING: Do *NOT* run the generated script without reviewing it carefully first!"
+
+# github markdown markers used to surround a code block. All text within the
+# markers is rendered in a fixed font.
+typeset -r block_open="\`\`\`bash"
+typeset -r block_close="\`\`\`"
+
+# convention used in all documentation to represent a non-privileged users
+# shell prompt. All lines starting with this value inside a code block are
+# commands the user should run.
+typeset -r code_prompt="\$ "
+
+usage()
+{
+	cat <<EOT
+Usage: ${script_name} <markdown-file>
+
+This script will convert a github-flavoured markdown document file into a
+bash(1) script to stout by extracting the bash code blocks.
```

Comment:
Is "stout" a misspelling here? I am assuming it should be "start" but...I won't assume :)

Reply:
Ha haaa! Good catch - that should be "stdout" (the standard abbreviation for "standard output").